custom_model_runner/CHANGELOG.md

```diff
@@ -4,6 +4,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+#### [1.12.0] - 2024-09-03
+##### Changed
+- Update support to Py>=3.8
```

Comment:
I'd say >= 3.9

Reply:
Doesn't work. 
Our R env is Py3.8.
Requires to invest a bit more time to update.